evennia/commands/default/muxcommand.py

```diff
@@ -109,15 +114,44 @@ def parse(self):
             else:
                 args = ""
                 switches = switches[0].split('/')
+            # Parse mux options, comparing them against user-provided switches, expanding abbreviations.
+            if hasattr(self, "options") and self.options and switches:
+                # If specific options are known, test them against given switches.
+                valid_switches, unused_switches, extra_switches = [], [], []
+                for element in switches:
+                    option_check = [each for each in self.options if each.lower().startswith(element.lower())]
+                    if len(option_check) > 1:
```

Comment:
Minor style thing would be to put `len(option_check)` into a variable and use that for comparison rather than having to do `len()` over and over. No big deal but it's good practice to re-use a result when possible.

Reply:
Done, committing an update soon.